test/system/data/required-packages-fedora-34

```diff
@@ -0,0 +1,3 @@
+git
```

Comment:
This will duplicate our lists of packages we want in the images... I'm not sure whether there's a better place for this, though.

Reply:
It will duplicate the extra packages, but the list is supposed to also include other packages that could be in the image by default. For example, the change they are about to do about utils-linux package.

Reply:
We need to make a list of what we consider the "essentials" of a toolbox image. If in the future any change escape to our attention, this will warn us.

Also I'm about to add a check for the existence of the required-packages file and if it does not exist, skip the test with a warning.